### ISSUE TITLE
[Bug] Noxesium crash with latest Indium

### ISSUE THREAD

**Issue by yichifauzi** [Noxcrew/noxesium#73]
Minecraft version : 1.20.6
here's crash logs
https://mclo.gs/Wfr6RQ7

**Comment by LobbyLC** [comment]
I was about to create a separate issue to report an incompatibility between Noxesium and Iris Shaders, though it seems to be directly related to this one so here's a little more info:

This message is sent in `latest.log` right before the game closes:
```
[Render thread/WARN]:
 @Redirect conflict. Skipping noxesium.mixins.json:beacon.BeaconRenderingMixin from mod noxesium->@Redirect::getGlobalBlockEntities(Lnet/minecraft/class_761;)Ljava/util/Set; with priority 1000, already redirected by mixins.iris.json:MixinLevelRenderer_SkipRendering from mod iris->@Redirect::skipGlobalBlockEntities(Lnet/minecraft/class_761;)Ljava/util/Set; with priority 1000
```
Here's my mods list on both 1.20.4 and 1.20.6:
```
Fabric API (Latest)
Iris Shaders (Latest)
Noxesium (Latest)
Sodium (Latest)
```
I'm getting the same crash as OP - it appears to be an issue with Iris Shaders rather than Indium.

**Comment by Aeltumn** [comment]
The same issue affects both, it's just the beacon rendering mixin that needs to be updated.

**Comment by Aeltumn** [comment]
Will be fixed in the update for 1.21.
